### ISSUE TITLE
Cannot load save game

### ISSUE THREAD

**Issue by suamor** [Ic0nic0de/TheCuratorsCompanion#21]
I don't know if this mod is the real cause, but so far the only hint that my game is no longer loading.
I recently updated LOTD, Curators Companion, Patches etc. Papyros Logging was disabled.

After update of two animation mods (Ostim, Immersive Interaction) as well as reinstalling a patch mod (QUASICPC - actually removed lightning-related patches because of EEL removal and update of Luminosity) and removal of more older Luminosity patches causing lighting issues I noticed that a new game was behaving very strange - some mods were not started.

I made several save games but only tried to load two of them so far. Loading of mods is even further delayed (even running game 8h does not finish scripts). 

What I see is that sometimes hair color not loading and wrong makeup applied, sometimes even skin color is incorrect.
When I reload the same game 1 or two times without restarting skyrim usually that goes away.
Other scripts never load.

I activated Paprys Logging and saw that after first time loading my save, my Papyros Log was bloatined (> 20 MB instead of previously < 10MB). As far as I see it there was only one function rerun all the time:

[05/22/2021 - 08:26:21AM] Error: Cannot call GetValue() on a None object, aborting function call
stack:
	[DBM_CuratorsCompanion (7A000D62)].rn_utility_script.RunSetup() - "RN_Utility_Script.psc" Line 193
	[DBM_CuratorsCompanion (7A000D62)].rn_utility_script.OnInit() - "RN_Utility_Script.psc" Line 136

It's happening so fast, that no other script has a chance to run. Any idea how to prevent this from happening? 
I struggle to find any possible cause of Curators behaviour after the mentioned changes above.

I will check if I can change some scanning settings (delayed or greatly slowed). If it does not help I fear I will have to disable this mod for now.

A great help would be how to find out possible conflicts with other mods in this situation. So far I only have Papyrus log identifying mods that slow down Papyrus.

**Comment by suamor** [comment]
First scan is never finished, even after 8h.

**Comment by suamor** [comment]
Several hints:
- Only occurs when save game is loaded
- Something is blocking something inside Curator to finish initialization
Curator is not the only mod, but for this case unfortantely the only very well visible.

I have found no systematic way to fix issue. I checked many things in my mod installation and found inconsistencies (mainly because mod manager does only uninstall original files, so there can be leftovers). It could be a script leftover or esp leftover that was blocking scripts progress. 

It's unfortunate that this unrelated mod (which I already supposed) has given me such a headache in analysis.

I will close the issue.
